index.js

```diff
@@ -1 +1,96 @@
-console.log('Hello, Node!')
+const fs = require('fs')
+const contents = fs.readFileSync('./data/pets.json', 'utf-8')
+const pets = JSON.parse(contents)
+
+const path = require('path')
+const petsFile = path.join(__dirname, 'data', 'pets.json')
+
+const read = () => {
+  return(pets)
```

Comment:
Should be reading and parsing file inside read function, otherwise will only contain a snapshot of the original file.